Helper/ImagePathResolver.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Metrilo\Analytics\Helper;
+
+class ImagePathResolver extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    /**
+     * Takes the best quality image url with timestamp of last modification
+     * @param $product / store product or quoteItem
```

Comment:
This won't work if you pass a quoteItem here, so this param is **always** product